Makefile

```
@@ -54,10 +54,11 @@ GO_SUBDIRS += cmd internal apis
 # ====================================================================================
 # Setup Kubernetes tools
 
-KIND_VERSION = v0.15.0
-UP_VERSION = v0.39.0
+KIND_VERSION = v0.30.0
+UP_VERSION = v0.41.0
 UP_CHANNEL = stable
-UPTEST_VERSION = v0.5.0
+UPTEST_VERSION = v2.0.1
```

```suggestion
UPTEST_VERSION = v2.1.0
```

suggestion(non-blocking): We might want to consider using the upcoming release that includes https://github.com/crossplane/uptest/pull/49 or we can bump it in a follow-up PR